### ISSUE TITLE
Implement online learning

### ISSUE THREAD

**Issue by georgerichardson** [Data4Democracy/internal-displacement#145]
As more articles are gathered, analysed and verified by a human, it would be nice for the ML models to self update. Open to discussion on tools and best practices for this.

**Comment by domingohui** [comment]
What are the ML models used for? I must have lost track in the slack convo...

**Comment by georgerichardson** [comment]
The ML I'm thinking of for this particular issue are to tag the articles as about displacement/not about displacement and then as disaster/conflict/other as the reason for displacement

**Comment by oskarsinger** [comment]
I have an extensive online learning background. Would be interested in details of the problem. Sounds like it may be better to just retrain periodically because this sounds like a supervised learning problem, but it depends on more specifics.

**Comment by emtseng** [comment]
Agreed @oskarsinger. Details pending, of course, but I imagine it may be useful to investigate and then set a threshold for the number of new articles tagged to trigger a retraining.
